src/typer.js

```diff
@@ -122,7 +122,10 @@
         if (scope.startTrigger !== undefined) {
           var unregister = scope.$watch('startTrigger', function(newVal, oldVal) {
             if (typeof newVal === 'boolean' && newVal) {
-              start(config, el);
+              $timeout(function() {
+                start(config, el);
+              }, config.startDelay);
```

Comment:
If the user does not set the `start-delay` property it defaults to 500ms. This may cause a slight delay for users who do not want a delay. 

To fix this you could change the default value of `config.startDelay` to 0 (line 73), so the trigger will happen almost instantaneously.